myhpi/templates/base.html

```diff
@@ -76,13 +81,20 @@
                 <div id="nav-level-2" class="nav-level"></div>
 
                 <div id="nav-level-right"
-                    class="nav-level d-flex flex-column flex-grow-1 align-items-stretch align-items-xl-end">
+                    class="nav-level d-flex flex-column flex-grow-2 align-items-stretch align-items-xl-end">
                     <ul id="nav-item-container-user" class="nav-item-container collapse"
                         data-navbar-level="#nav-level-0">
+                        {% if user.groups.all|length > 0 %}<span class="nav-bar-group-header"><i>Deine Gruppen:</i></span>{% endif %}
+                        {% for group in user.groups.all %}
+                            <li class="nav-bar-group-item">{{ group }} </li>
+                            {% empty %}
```

Comment:
what do you need the empty for?

Reply:
Removed in update. 
If we want to display something along the lines of "You're not part of any group yet, join now" etc. in the future we could use empty.